### PR TITLE
Add Fedora 24 image mappings

### PR DESCRIPTION
**Pull request by mellinoe**
@mmitche

### CODE REVIEW

**Comment by mmitche** [comment]
@mellinoe We'll need to move these images into the new deployment model (it's simple).  I'll walk you through it tomorrow.

File: jobs/generation/Utilities.groovy [hunk 1]
```diff
@@ -300,6 +300,13 @@ class Utilities {
                                 // For outerloop runs
                                 'outer-latest-or-auto':'auto-fedora23-20160622outer'
                                 ],
+                           'Fedora24' :
+                                [
+                                // Latest auto image.
+                                'latest-or-auto':'auto-fedora24-20161024',
```
Comment (mmitche) [inline jobs/generation/Utilities.groovy]: can you drop the 'auto-' bit of this?

File: jobs/generation/Utilities.groovy [hunk 2]
```diff
@@ -300,6 +300,13 @@ class Utilities {
...
+                                // For outerloop runs
+                                'outer-latest-or-auto':'auto-fedora24-20161024outer'
```
Comment (mmitche) [inline jobs/generation/Utilities.groovy]: can you drop the auto- and add a - before outer?

**Comment by mellinoe** [comment]
@mmitche How does this look now?

**Comment by mmitche** [comment]
LGTM

**Pull request merged**